src/cli/dev.test.ts

```diff
@@ -131,7 +131,7 @@ describe("dev command", () => {
       await cb({ errors: ["error"] });
     });
 
-    await program.parseAsync(["node", "test", "dev", "--yes"]);
+    await program.parseAsync(["npx", "pepr", "dev", "--yes"]);
```

Comment:
```suggestion
    await program.parseAsync(["dev", "--yes"], { from: "user" });
```
Consistency with other tests we've written, though it's functionally equivalent